### PR TITLE
Add documentation articles for Laravel and .NET

### PR DESCRIPTION
**Pull request by mabryl**
Adding new guides to our Integrations docs on how the integrate the editor with Laravel and .NET.

### CODE REVIEW

File: docs/installation/integrations/dotnet.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+````
+├── bin
+├── obj
+├── Pages
+├── Properties
+├── wwwroot
+│   ├── assets
+|      ├── vendor
+|          └── ckeditor5
```
Comment (michalbilik) [inline docs/installation/integrations/dotnet.md:54]: While following the guide I did not encounter any bugs, but one small issue may arise when the user downloads the .zip from online builder. The end user needs to change the folder name or change the script’s path. For instance, the original folder name "/ckeditor5-38.0.1-8s0tbeh90pmq" should be changed to "/ckeditor5" for the project to function correctly.

File: docs/installation/integrations/dotnet.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    <h1>Welcome to CKEditor 5 in .NET</h1>
+    <div id="editor"></div>
+    <script src="assets/vendor/ckeditor5/build/ckeditor.js"></script>
+    <script>
+        ClassicEditor
+            .create( document.querySelector( '#editor' ) )
+            .catch( error => {
+                console.error( error );
+            } );
+    </script>
```
Comment (michalbilik) [inline docs/installation/integrations/dotnet.md:83]: If a new user prepares a build different from ClassicEditor he needs to change the script accordingly to the chosen build.

File: docs/installation/integrations/laravel.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+````
+├── app
+├── bootstrap
+├── config
+├── database
+├── public
+│   ├── assets
+|      ├── vendor
+|          └── ckeditor5
```
Comment (michalbilik) [inline docs/installation/integrations/laravel.md:55]: While following the guide I did not encounter any bugs, but one small issue may arise when the user downloads the .zip from online builder. The end user needs to change the folder name or change the script’s path. For instance, the original folder name "/ckeditor5-38.0.1-8s0tbeh90pmq" should be changed to "/ckeditor5" for the project to function correctly.

File: docs/installation/integrations/laravel.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+<body>
+    <h1>Welcome to CKEditor 5 in Laravel</h1>
+    <div id="editor"></div>
+    <script>
+        ClassicEditor
+            .create( document.querySelector( '#editor' ) )
+            .catch( error => {
+                console.error( error );
+            } );
+    </script>
```
Comment (michalbilik) [inline docs/installation/integrations/laravel.md:85]: If a new user prepares a build different from ClassicEditor he needs to change the script accordingly to the chosen build.

**Review by godai78 — APPROVED** [review]
LGTM

File: docs/installation/integrations/dotnet.md (hunk 1 above)
Comment (mabryl) [reply docs/installation/integrations/dotnet.md:54]: Good point! I included an `info-box` that clarifies this step.

File: docs/installation/integrations/dotnet.md (hunk 2 above)
Comment (mabryl) [reply docs/installation/integrations/dotnet.md:87]: Another good point! I included another `info-box` that clarifies this step.

File: docs/installation/integrations/laravel.md (hunk 4 above)
Comment (mabryl) [reply docs/installation/integrations/laravel.md:89]: Another good point! I included another info-box that clarifies this step.

File: docs/installation/integrations/laravel.md (hunk 3 above)
Comment (mabryl) [reply docs/installation/integrations/laravel.md:55]: Good point! I included an info-box that clarifies this step.

**Comment by mabryl** [comment]
Added `info-boxes` in accordance with @michalbilik's insights. This should clarify the integration steps a little bit. Let me know what you think @godai78 @apadol-cksource.

**Review by michalbilik — APPROVED** [review]
LGTM

**Review by godai78 — APPROVED** [review]
LGTM

**Review by apadol-cksource — CHANGES_REQUESTED** [review]

**Comment by mabryl** [comment]
Thank you for your insights @apadol-cksource. I added the requested changes.

**Review by apadol-cksource — APPROVED** [review]

**Pull request merged**